File: Examples/VisualBasic/ApplyLicense/ApplyLicenseByPath.vb
Imports Aspose.Diagram
Imports System

Public Class ApplyLicenseByPath
    Public Shared Sub Run()
        ' ExStart:ApplyLicenseByPath
        ' set path of the license file, i.e. c:\temp\
        Dim dataDir As String = "c:\temp\"

        Dim license As New License()
        license.SetLicense(dataDir & Convert.ToString("Aspose.Diagram.lic"))
        ' ExEnd:ApplyLicenseByPath
    End Sub
End Class

File: Examples/CSharp/ApplyLicense/ApplyLicenseByPath.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Diagram.Examples.CSharp.ApplyLicense
{
    public class ApplyLicenseByPath
    {
        public static void Run() 
        {
            // ExStart:ApplyLicenseByPath
            // set path of the license file, i.e. c:\temp\
            string dataDir = @"c:\temp\";

            License license = new License();
            license.SetLicense(dataDir + "Aspose.Diagram.lic");
            // ExEnd:ApplyLicenseByPath
        }
    }
}
